test/dutchExchange-claimSellerFunds.js

```diff
@@ -0,0 +1,124 @@
+
+/* eslint no-console:0, max-len:0, no-plusplus:0, no-mixed-operators:0, no-trailing-spaces:0 */
+/* tulip issuing is tested seperately in dutchExchange-Tulip.js */
+
+const { 
+  eventWatcher,
+  assertRejects,
+} = require('./utils')
+
+const {
+  setupTest,
+  getContracts,
+  getAuctionIndex,
+  waitUntilPriceIsXPercentOfPreviousPrice,
+  setAndCheckAuctionStarted,
+  postBuyOrder,
+  postSellOrder,
+} = require('./testFunctions')
+
+// Test VARS
+let eth
+let gno
+let dx
+
+let contracts
+
+const setupContracts = async () => {
+  contracts = await getContracts();
+  // destructure contracts into upper state
+  ({
+    DutchExchange: dx,
+    EtherToken: eth,
+    TokenGNO: gno,
+  } = contracts)
+}
+const startBal = {
+  startingETH: 90.0.toWei(),
+  startingGNO: 90.0.toWei(),
+  ethUSDPrice: 1008.0.toWei(),
```

Comment:
this may throw the tests with the new `require` add-ins - I've been using `1100`